ROLES.md

```diff
@@ -204,8 +214,14 @@ If a maintainer becomes inactive in the project for an extended period of time,
 emeritus maintainer. Emeritus maintainers lose their ability to approve code contributions, but retain their voting rights
 for up to one year. After one year, emeritus maintainers revert back to being normal members. 
 
-A maintainer becomes an emeritus maintainer if the individual hasn't contributed a PR to the project in 3 months. An emeritus 
-maintainer can regain full maintainer status by submitting 5 PRs to the project.
+Contributing to the project can be done in a number of ways:
+
+- Merging pull requests affecting one of the maintained paths
+- Triaging issues or pull requests in a maintained repo
+- Commenting on issues or pull requests in a maintained repo
```

Comment:
is commenting on issues sufficient for maintained repo for maintainers to maintain status?

Reply:
I suggest removing commenting so it is somewhat higher requirement than members.

Reply:
Done.